docs/COMMITTER_GUIDE.md

```diff
@@ -0,0 +1,41 @@
+<!--
+Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the GPL, Version 3.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# Committer Guidelines
+
+These are the guidelines for people with commit privileges on the GitHub repository. Committers act as members of the Core Team and not necessarily employees of Dell.
+
+These guidelines apply to everyone and as Committers you have been given access to commit changes because you exhibit good judgment and have demonstrated your commitment to the vision of the project. We trust that you will use these privileges wisely and not abuse it.
+
+If these privileges are abused in any way and the quality of the project is compromised, our trust will be diminished and you may be asked to not commit or lose these privileges all together.
+
+## General Rules
+
+### Don't
+
+* Break the build.
```

Comment:
This is not applicable to our repo. There is no "build" per-se. This item could be replaced with one that has similar intent. For example:

Introduce Ansible sanity regression failures.

It would also be worthwhile to add a bullet under "Don't" for:

Introduce enterprise_sonic regression test failures.

Reply:
Okay, we will remove the build bullet. We will discuss with OneAnsible technical team, and then add bullets for "sanity failures" and "regression failures".

Reply:
@kerry-meyer  We have updated the above comments as suggested. kindly review and approve the PR.